lib/logger/build.sbt

```diff
@@ -0,0 +1,7 @@
+libraryDependencies ++=
+  Seq(
+//      "com.github.pureconfig" %% "pureconfig" % "0.9.0"
```

Comment:
we are not going to use pureconfig, it has all the flaws of original typesafe-config